internal/security/gnxi/rpcs.go

```diff
@@ -0,0 +1,1078 @@
+// This following provide a list of all RPCs related to FP testing.
+// The below code is generated using ../gen/generate.go. Please do not modify.
+package gnxi
+
+type rpcs struct {
+	ALL                                                                   *RPC
```

Comment:
Go style is to _not_ put these in all caps. Also, do the fields need to be exported here?